src/marten/core/sluggable.cr

```diff
@@ -0,0 +1,31 @@
+module Marten
+  module Core
+    # The Sluggable module provides functionality for generating URL-friendly slugs from strings.
+    module Sluggable
+      extend self
```

Comment:
Why use `extend self` here? I would expect that abstractions requiring slug generation capabilities would simply include `Marten::Core::Sluggable`.